### ISSUE TITLE
fix 31th anniversary

### ISSUE THREAD

**Issue by kenirwin** [kenirwin/Then-And-Now-Photos#1]
* right now it assumes that all year ranges take a -th; allow -st, -nd, -rd

**Comment by kenirwin** [comment]
check out: https://www.php.net/manual/en/numberformatter.format.php
```
<?php
    // Needs 'php5-intl' package to be installed on Debian/Ubuntu
    $set_format = numfmt_create( 'en_US', NumberFormatter::ORDINAL );
    // '3' is displayed as '3rd'
    echo numfmt_format( $set_format, 3 );
?>
```

**Comment by kenirwin** [comment]
Fixed in 95e7772
